operate/services/manage.py

```diff
@@ -1297,7 +1297,7 @@ def _get_current_staking_program(
                 .functions.getStakingState(service_id)
                 .call()
             )
-        except web3.exceptions.ContractLogicError:
+        except Exception:  # pylint: disable=broad-except
```

Comment:
The same comment applies here: https://github.com/valory-xyz/olas-operate-middleware/pull/44/files#r1995345198